future-apps/payroll/contracts/Payroll.sol

```diff
@@ -557,8 +559,7 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
      */
     function _setRateExpiryTime(uint64 _time) internal {
         // Require a sane minimum for the rate expiry time
-        // (1 min == ~4 block window to mine both a pricefeed update and a payout)
-        require(_time > 1 minutes, ERROR_EXPIRY_TIME_TOO_SHORT);
+        require(_time >= MIN_RATE_EXPIRY, ERROR_EXPIRY_TIME_TOO_SHORT);
```

Comment:
It seems more clear that "minimum" would be loose (you can choose a value greater than or equal to the minimum). This is also the definition we use for "minimum" for the Finance app's periods.